packages/playground/src/utils/nodeSelector.ts

```diff
@@ -224,6 +224,9 @@ export async function validateRentContract(
   }
 
   try {
+    if (node.dedicated || node.rentedByTwinId === 0) {
```

Comment:
i think it should be and 
I'm already reting node 151 but got this error
<img width="877" alt="image" src="https://github.com/user-attachments/assets/85fdd079-3b28-407e-a5bb-c6b0b1465c45" />